src/handlers/userHandlers.js

```diff
@@ -0,0 +1,63 @@
+import axios from 'axios';
+import { pathObj } from './../utils/generalVariables';
+
+export const getUserHandler = (token, userId) => {
+  if (!userId || typeof userId !== "string") {
+    return new Error("Make sure you're passing a valid username address and a password that's at least 8 characters long")
```

Comment:
Here you are not validating any length so I would even remove that part of the error, also in line 27

Reply:
See above